safekeeper/src/lib.rs

```diff
@@ -99,6 +106,10 @@ pub struct SafeKeeperConf {
     pub peer_recovery_enabled: bool,
     pub remote_storage: Option<RemoteStorageConfig>,
     pub max_offloader_lag_bytes: u64,
+    /* BEGIN_HADRON */
+    pub max_reelect_offloader_lag_bytes: u64,
```

Comment:
~~those field should be optional otherwise breaks compatibility?~~

okay I realized that default value 0 == disabled